src/App.jsx

```diff
@@ -14,70 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [product, setProduct] = useState('Jam');
+  const NO_GOODS_SELECTED = 'No goods';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {product}
+        {product !== NO_GOODS_SELECTED
+          ? ' is '
+          : ' '
+        }
+        selected
```

Comment:
Try to refactor this. If you have selected a product then show this info, otherwise just show "No goods selected message"